src/software/ai/play_selection_fsm.h

```diff
@@ -63,108 +53,114 @@ struct PlaySelectionFSM
 
     /**
      * Action to set up the OverridePlay, SetPlay, StopPlay, HaltPlay,
-     * OffensivePlay, or DefensivePlay
+     * OffensePlay, or DefensePlay
      *
      * @param event The PlaySelection::Update event
      */
-    void setupOverridePlay(Update event);
     void setupSetPlay(const Update& event);
     void setupStopPlay(const Update& event);
     void setupHaltPlay(const Update& event);
-    void setupOffensivePlay(const Update& event);
-    void setupDefensivePlay(const Update& event);
+    void setupOffensePlay(const Update& event);
+    void setupDefensePlay(const Update& event);
+
+    /**
+     * Action to reset the current SetPlay to none
+     *
+     * @param event The PlaySelection::Update event
+     */
+    void resetSetPlay(const Update& event);
 
     /**
-     * Action to evaluate the currently running DynamicPlay
+     * Action to terminate the OffensePlay
      *
      * @param event The PlaySelection::Update event
      */
-    void evaluateDynamicPlay(const Update& event);
+    void terminateOffensePlay(const Update& event);
 
     auto operator()()
     {
         using namespace boost::sml;
 
-        DEFINE_SML_STATE(Halt)
-        DEFINE_SML_STATE(Stop)
-        DEFINE_SML_STATE(SetPlay)
-        DEFINE_SML_STATE(OffensivePlay)
-        DEFINE_SML_STATE(DefensivePlay)
+        DEFINE_SML_STATE(HaltState)
+        DEFINE_SML_STATE(StopState)
+        DEFINE_SML_STATE(SetPlayState)
+        DEFINE_SML_STATE(OffensePlayState)
+        DEFINE_SML_STATE(DefensePlayState)
```

Comment:
the `_S` implies `State` but that's a nit

Reply:
OffensePlay as a state would conflict with the actual OffensePlay, so I added State to the end. Then I added State at the end for all of them to be consistent